sources/models/Cargo.toml

```diff
@@ -11,22 +11,14 @@ exclude = ["README.md"]
 
 [dependencies]
 bottlerocket-release = { path = "../bottlerocket-release", version = "0.1" }
+# settings plugins
+bottlerocket-settings-models = { workspace = true }
+bottlerocket-settings-plugin = { workspace = true }
```

Comment:
I wonder if this would be clearer if it remained at the bottom instead of being alphabetical.